src/workload_api/client.rs

```diff
@@ -319,16 +318,10 @@ impl WorkloadApiClient {
         &self,
         audience: T,
         jwt_token: &str,
-    ) -> Result<(SpiffeId, Option<Claims>), ClientError> {
-        let response = self.validate_jwt(audience, jwt_token)?;
-
-        let claims = response
-            .claims
-            .into_option()
-            .map(|claims| claims.try_into())
-            .transpose()?;
-
-        Ok((response.spiffe_id.try_into()?, claims))
+    ) -> Result<JwtSvid, ClientError> {
+        self.validate_jwt(audience, jwt_token)?;
```

Comment:
Maybe you could use `let _ = self.validate_jwt(...)?;` with a comment above explaining it's fine to ignore the spiffe ID and claims returned?

I am wondering why those are returned at all in the first place 🤔